.github/workflows/push-master.yml

```diff
@@ -4,17 +4,23 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - master
 
 jobs:
   build_test_publish:
-    name: Build, test, and publish unstable release
-    if: "contains(github.event.head_commit.message, 'chore(release): publish')"
```

Comment:
Do we still want this `if` check to avoid trying to release something that's alredy released?